distributed/widgets/templates/client.html.j2

```diff
@@ -0,0 +1,47 @@
+<div>
```

Comment:
I think we need to include an entry to our [`MANIFEST.in` file](https://github.com/dask/distributed/blob/ac55f25d230c7144ed618d1d4374f254303a4e0a/MANIFEST.in) for these templates so they are packaged with `distributed`

Reply:
Good point, added all `.j2` files to the manifest.